apps/cookbook/src/app/examples/card-example/examples/card-example.shared.scss

```
@@ -79,3 +84,27 @@ button {
     transform: scale(0.95);
   }
 }
+
+.variant-card-container {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 16px;
+  @container (width < 616px) {
+    flex-direction: column;
+    align-items: center;
+
+    kirby-card {
+      max-width: none !important;
+      width: 100%;
+    }
+  }
+
+  kirby-card {
+    max-width: 50%;
+    min-width: 250px;
+    flex: 1;
+  }
```

Consider reordering rulesets to avoid `!important` flag and have default styles declared first.
Also use flex to control width of cards:
```suggestion
  kirby-card {
    max-width: 50%;
    min-width: 250px;
    flex: 1;
  }
  
  @container (width < 616px) {
    flex-direction: column;
    align-items: stretch;

    kirby-card {
      max-width: initial;
    }
  }


```